File: 240621/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.py
find_chr = input()

five_strs = ["apple", "banana", "grape", "blueberry", "orange"]

answer = 0
for one_str in five_strs:
    if one_str[2] == find_chr or one_str[3] == find_chr:
        print(one_str)    
        answer += 1

print(answer)

File: 250910/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <iostream>
#include <vector>
#include <string>
using namespace std;

int main() {
    // Please write your code here.
    char f_chr;
    cin >> f_chr;
    
    vector<string> fives = {"apple", "banana", "grape", "blueberry", "orange"};
    vector<string> ans_lst;
    int answers = 0;

    for(const auto& word : fives)
    {
        if(word[2] == f_chr || word[3] == f_chr)
        {
            ans_lst.push_back(word);
            answers++;
        }
    }

    for(const auto& ans : ans_lst)
    {
        cout << ans << endl;
    }

    cout << answers << endl;
    return 0;
}